charts/oes/config/oes-platform/platform-local.yml

```diff
@@ -42,7 +34,22 @@ ldap:
   groupSearchFilter: {{ .Values.global.ldap.groupSearchFilter }}
   groupRoleAttributes: {{ .Values.global.ldap.groupRoleAttributes }}
   userDnPattern: {{ .Values.global.ldap.userDnPattern }}
-
+dashboard:
+  scheduler:
+    enable: false
+    fixedDelay:
+      in:
+        milliseconds: 120000
+    initialDelay:
+      in:
+        milliseconds: 300000
+    workerThreads: 50
+database:
+  cleanup:
+    scheduler:
+      enabled: {{ .Values.dbcleanup.enabled }}
```

Comment:
@girichinna27  this will be enabled or disabled ?? Are we changing anything here